src/main/scala/no/ndla/imageapi/controller/HealthController.scala

```diff
@@ -8,15 +8,50 @@
 
 package no.ndla.imageapi.controller
 
-import org.scalatra.{Ok, ScalatraServlet}
+import no.ndla.imageapi.ImageApiProperties
+import no.ndla.imageapi.model.api.{ImageMetaSummary, SearchResult}
+import org.json4s._
+import org.json4s.native.JsonParser
+import org.scalatra._
+
+import scalaj.http.{Http, HttpResponse}
 
 trait HealthController {
   val healthController: HealthController
 
   class HealthController extends ScalatraServlet {
 
+    implicit val formats = DefaultFormats
+
+    def getApiResponse(url: String): HttpResponse[String] = {
+      Http(url).execute()
+    }
+
+    def getImageUrl(body: String): (Option[String], Long) = {
+      val json = JsonParser.parse(body).extract[SearchResult]
+      json.results.headOption match {
+        case Some(result: ImageMetaSummary) => (Some(result.metaUrl), json.totalCount)
+        case _ => (None,json.totalCount)
+      }
+    }
+
+    def getReturnCode(imageResponse: HttpResponse[String]): ActionResult = {
+      imageResponse.code match {
+        case 200 => NoContent()
```

Comment:
Nå begynner det å bli bra. Bare en liten ting til. `/health` kan kun returnere 200 for at komponenten skal ansees for å være "healthy". Både [AWS](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-healthchecks.html#health-check-configuration) forutsetter det og [verify_depoyment](https://kode.knowit.no/projects/NDLA/repos/deploy/browse/config/codedeploy/scripts/verify_deployment.sh#7) forutsetter dette.